docs/src/man/bootc-install-to-disk.md

```diff
@@ -96,7 +96,7 @@ disabled but where the target does have SELinux enabled.
 
 **\--karg**=*KARG*
 
-:   Add a kernel argument
+:   Add a kernel argument (can be given multiple times)
```

Comment:
These are auto-generated right now, so we need to change the Rust code, and then run `cargo xtask manpages`.

It seems like the cargo functionality for this could be improved, because the Rust side definitely knows it can be passed multiple times.